GitDiffMargin/GitDiffMargin.csproj

```diff
@@ -118,8 +117,8 @@
     <Compile Include="Core\WeakReference.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="$(NuGetPackageRoot)\libgit2sharp.nativebinaries\2.0.306\runtimes\win-x86\native\git2-106a5f2.dll">
-      <Link>x86\git2-106a5f2.dll</Link>
+    <Content Include="$(PkgLibGit2Sharp_NativeBinaries)\runtimes\win-x86\native\$(libgit2_filename).dll">
+      <Link>x86\$(libgit2_filename).dll</Link>
```

Comment:
Where is the `libgit2_filename` defined?

Reply:
Found it 
![image](https://user-images.githubusercontent.com/272612/111035559-63e70500-841b-11eb-96cd-223912bf93a5.png)